src/services/mongoose/accounts.ts

```diff
@@ -76,25 +104,60 @@ export const AccountsRepository = (): IAccountsRepository => {
         username: account.username as Username,
         mapInfo: {
           title: account.title as BusinessMapTitle,
-          coordinates: account.coordinate as Coordinates,
+          coordinates: account.coordinates as Coordinates,
         },
       }))
     } catch (err) {
       return new UnknownRepositoryError(err)
     }
   }
+
+  // currently only used by Admin
+  const update = async ({
```

Comment:
The comment could be encoded in the function name:
```
updateFromAdminUseCase
```

Not a blocker - just a thought.